tests/integration/test_integration.py

```diff
@@ -79,3 +74,28 @@ async def test_relate_and_wait_for_active_status(
         status="active",
         timeout=1000,
     )
+
+
+@pytest.mark.abort_on_fail
+async def test_remove_nrf_relation_and_wait_for_blocked_status(
```

Comment:
I think  test_remove_nrf_and_wait_for_blocked_status as test name makes more sense as we are removing NRF. Consequently NRF relation is removed. 

We can add another test just removing the relation and while keeping NRF deployed.

Reply:
Updated.